src/parser/jobs/drg/modules/BloodOfTheDragon.js

```diff
@@ -54,6 +60,37 @@ export default class BloodOfTheDragon extends Module {
 		this.addHook('complete', this._onComplete)
 	}
 
+	// duplicate code from other PRs
```

Comment:
I'm assuming this is going to get tidied up/consolidated once everything is merged?

Reply:
that's the plan, just marking it with a comment to remember which functions are duplicated.